Vagrantfile

```diff
@@ -0,0 +1,30 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "ubuntu/focal64"
+  config.vm.provision "shell", privileged: false, inline: <<-SHELL 
+    sudo apt-get update
+    sudo apt-get upgrade -y
+    sudo apt-get autoremove -y
+    sudo apt-get install -y python3 python3-pip python3-virtualenv
```

Comment:
I know I'm deviating from the instructions by not using `pyenv` -- but personally I prefer to avoid using additional tools where they're not necessary. Python 3.8 is perfectly adequate for our needs right now.

Reply:
Sure, that's fair enough, but note that you are not installing Python 3.8.

You are installing the latest Python 3.x, and releases are not all backwards compatible. You should pin it to a specific version.  I believe you can `apt-get install python3.8`

Reply:
You're right 👍 

With Ubuntu you wouldn't actually end up with Python 3.9+ until you replace focal with the next one (Hirsute?). So fair enough.

Also backwards incompatibilities in Python are rare. This was just a very minor point because I like to be explicit about versions.